app/src/main/java/openfoodfacts/github/scrachx/openfood/views/SaveProductOfflineActivity.java

```diff
@@ -147,7 +147,7 @@ protected void onCreate(Bundle savedInstanceState) {
         } else {
             mProduct = new SendProduct();
             mProduct.setBarcode(mBarcode);
-			barcodeText.setText(barcodeText.getText() + " " + mBarcode);
+			barcodeText.setText(getString(R.string.format_barcode_text, barcodeText.getText(),  mBarcode));
```

Comment:
I don't understood the advantage to put the space character in the strings.xml as a string template.
The current code is much more clearer.
what do you think @itchix ?